src/app/scripts/cac/control/cac-control-itinerary-list.js

```diff
@@ -17,7 +17,22 @@ CAC.Control.ItineraryList = (function (_, $, MapTemplates) {
             alert: '.alert',
             container: '.directions-list',
             hiddenClass: 'hidden',
+            itineraryList: '.routes-list',
             itineraryItem: '.route-summary'
+        },
+        // Settings for 'slick' carousel for swiping itineraries on mobile
+        slick: {
+            arrows: false,
+            dots: true,
+            infinite: false,
+            mobileFirst: true,
+            variableWidth: true,
+            responsive : [
+                {
+                    breakpoint: 480,
```

Comment:
Would be good to note here and in `_breakpoints.scss` that this magic number needs to match, so it doesn't get changed in one place but not the other in future.